rust_src/src/lisp.rs

```diff
@@ -34,50 +37,30 @@ use marker::{LispMarker, marker_position};
 ///
 /// Their definition are determined in a way consistent with Emacs C.
 /// Under casual systems, they're the type isize and usize respectively.
-
-include!(concat!(env!("OUT_DIR"), "/definitions.rs"));
-/// These are an example of the casual case.
-#[cfg(dummy = "impossible")]
-pub type EmacsInt = isize;
-#[cfg(dummy = "impossible")]
-pub type EmacsUint = usize;
-#[cfg(dummy = "impossible")]
-pub type EmacsDouble = f64;
-#[cfg(dummy = "impossible")]
-pub const EMACS_INT_MAX: EmacsInt = 0x7FFFFFFFFFFFFFFF_i64;
-#[cfg(dummy = "impossible")]
-pub const EMACS_INT_SIZE: EmacsInt = 8;
-#[cfg(dummy = "impossible")]
-pub const EMACS_FLOAT_SIZE: EmacsInt = 8;
-#[cfg(dummy = "impossible")]
-pub const GCTYPEBITS: EmacsInt = 3;
-#[cfg(dummy = "impossible")]
-pub const USE_LSB_TAG: bool = true;
-
 #[repr(C)]
 #[derive(PartialEq, Eq, Clone, Copy)]
-pub struct LispObject(EmacsInt);
+pub struct LispObject(Lisp_Object);
```

Comment:
Also i came up with another idea for #138, maybe we could have a lint that detects when a function has a `Lisp_Object` in the parameters and doesn't has a `extern "C"` marker.